bin/update-plugins.py

```
@@ -0,0 +1,31 @@
+import argparse
+import subprocess
+import os
+import json
+
+parser = argparse.ArgumentParser(
+    prog="update-plugins",
+    description="Update all plugins in the flake")
+
+
+LOCAL_PINS = "npins/sources.json"
+PINS = os.path.join(os.getcwd(), local_pins)
```

Ruff (0.8.2)

Undefined name `local_pins`

(F821)